launch/dynamo-run/src/lib.rs

```diff
@@ -41,6 +42,9 @@ const ENDPOINT_SCHEME: &str = "dyn://";
 /// the command line. Hence it's optional, and defaults to this.
 const INVISIBLE_MODEL_NAME: &str = "dynamo-run";
 
+/// The component name for the KV publisher, if used
+const KV_PUBLISHER_COMPONENT: &str = "kvpublisher";
```

Comment:
We'll probably need to reconcile our naming scheme for utility things like metrics. I think the python bindings have a hard-coded endpoint called "load_metrics" when using the publisher, and the component is either arbitrary or copied from the worker's component name in our current python example of such. 

Similarly here, when run via dynamo-run, the component will be a hard-coded thing of "kvpublisher".

For the `metrics` binary (or whatever means of getting observability from a dynamo deployment), it's gonna need some improvements soon to be more automatic in discovering things that are publishing metrics, so coming up with some consistent approaches or naming schemes will be useful there.

Not a request for change here, just food for thought in the near term.